tox.ini

```diff
@@ -46,12 +46,11 @@ commands =
 
 [testenv:py3-server]
 description = Runs all Python3-based server unit and functional tests
-setenv =
-    PYTHONPATH = {env:PYTHONPATH}:{toxinidir}/server/lib:{toxinidir}/lib
 deps =
     -r{toxinidir}/test-requirements.txt
     -r{toxinidir}/server/requirements.txt
 commands =
+    python -c "print('PYTHONPATH === ', PYTHONPATH)"
```

Comment:
I know this is gone from the latest commit, but I was skimming back through the recent commits out of curiosity.

Not sure what this is supposed to do, but `PYTHONPATH` isn't a Python variable so I imagine it didn't work?

If you really want to see the actual Python module search path, you can either use

```
python -c 'import sys; print(sys.path)`
````

or, for a nicer (but much larger) display,

```
python -m site
```

Reply:
See [this comment](https://github.com/distributed-system-analysis/pbench/pull/2414#discussion_r705413896) -- you even got credit, Dave!  😁

Reply:
I had even seen your comment, but apparently hadn't read it in sufficient detail. 😬